doc/rados/configuration/filestore-config-ref.rst

```diff
@@ -19,90 +24,96 @@ though Filestore OSDs are still supported.
 Extended Attributes
 ===================
 
-Extended Attributes (XATTRs) are important for Filestore OSDs.
-Some file systems have limits on the number of bytes that can be stored in XATTRs. 
-Additionally, in some cases, the file system may not be as fast as an alternative
-method of storing XATTRs. The following settings may help improve performance
-by using a method of storing XATTRs that is extrinsic to the underlying file system.
-
-Ceph XATTRs are stored as ``inline xattr``, using the XATTRs provided
-by the underlying file system, if it does not impose a size limit. If
-there is a size limit (4KB total on ext4, for instance), some Ceph
-XATTRs will be stored in a key/value database when either the
+Extended Attributes (XATTRs) are important for Filestore OSDs. However, Certain
+disadvantages can occur when the underlying file system is used for the storage
+of XATTRs: some file systems have limits on the number of bytes that can be
+stored in XATTRs, and your file system might in some cases therefore run slower
+than would an alternative method of storing XATTRs. For this reason, a method
+of storing XATTRs extrinsic to the underlying file system might improve
+performance. To implement such an extrinsic method, refer to the following
+settings.
+
+If the underlying file system has no size limit, then Ceph XATTRs are stored as
```

Comment:
I think we can remove this whole section.  If not, I think only EXT4 has the XATTRs issue.

Reply:
@neha-ojha @jdurgin: What say you?